### PR TITLE
Update README.adoc for better clarity 

### PR DESCRIPTION
**Pull request by DannyS03**
https://github.com/OpenLiberty/guide-rest-intro/issues/93

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -180,18 +180,20 @@ include::finish/src/main/liberty/config/server.xml[]
...
+. Configures the server to run the produced web application on a context root, specified in the [hotspot=webApplication file=0]`<webApplication/>` element. The produced web application is essentially the RESTful web service we attempt to create in this guide, which entails all the necessary configuration and utilizes Maven's project object model `pom.xml` file to manage the project's build. The [hotspot=appContextRoot file=1]`</appContextRoot>` is fed through in the `pom.xml` as property to also act as a value placeholder. 
 
 pom.xml
 [source,xml,linenums,role="code_column"]
 ----
 include::finish/pom.xml[]
 ----
 
-The variables that are being used in the [hotspot=httpEndpoint hotspot=webApplication file=0]`server.xml` file are provided by the properties set in the Maven [hotspot=defaultHttpPort hotspot=defaultHttpsPort hotspot=appContextRoot file=1]`pom.xml` file. The properties must be formatted as `liberty.var.variableName`.
+Ultimately, the server configuration is the only mandatory configuration file, and it's only requirement is that the root element is `</server>`, where optional configuration of files and parameters is based on user needs.
```
Comment (yeekangc) [inline README.adoc]: What is the purpose of this paragraph?

**Comment by yeekangc** [comment]
Take a look at what is here: https://github.com/OpenLiberty/guide-rest-intro/tree/singlepane

Reverting to what's there is a good and reasonable starting point.

File: README.adoc (hunk 1 above)
Comment (DannyS03) [reply README.adoc]: Right, excessive information.

**Comment by yeekangc** [comment]
@DannyS03, can you provide a summary of the latest changes?

**Comment by DannyS03** [comment]
> @DannyS03, can you provide a summary of the latest changes?


- Applied your feedback on the initial commits. 

### Configuring the server
image


### Testing the service
image

**Comment by gkwan-ibm** [comment]
Will address #93 by other PR

**Pull request closed without merging**